core/isolation/kit-adapter.php

```diff
@@ -12,4 +13,14 @@ public function get_kit_settings() {
 	public function get_main_post() {
 		return Plugin::$instance->kits_manager->get_kit_for_frontend()->get_main_post();
 	}
+
+	public function is_active_kit_default() : bool {
+		$kit_id = Plugin::$instance->kits_manager->get_active_id();
+
+		if ( false === $kit_id || null === $kit_id ) {
+			return false;
+		}
+
+		return esc_html__( 'Default Kit', 'elementor' ) === get_post( $kit_id )->post_title;
```

Comment:
Should we make sure `post_title` exists?

Reply:
Are you not going to add active_kit === previous_kit?

Reply:
We checked this and it's is redundant. Could we have missed something?